src/pages/Lend/Filter/FilterComponents/FilterRangeSlider.vue

```diff
@@ -8,141 +8,92 @@
 				({{ rangeLabel }})
 			</span>
 		</div>
-		<div class="slider">
+		<div
+			class="slider"
+			v-if="range.min !== range.max"
```

Comment:
does removing and re-injecting this from/to the DOM cause any weird re-renders?

Reply:
Not that I could find, do you think v-show would be better?